test/suites/integration/repair_policy_test.go

```diff
@@ -77,6 +77,10 @@ var _ = Describe("Repair Policy", func() {
 			Type:               corev1.NodeReady,
 			Status:             corev1.ConditionFalse,
 			LastTransitionTime: metav1.Time{Time: time.Now().Add(-31 * time.Minute)},
+		}, corev1.NodeCondition{
```

Comment:
This doesn't look like. I'm assuming that you're intending to create another `Entry` here -- looks like you might have attached this to the existing `Entry`